sensu/plugins/check-disk-by-mount.rb

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env /opt/sensu/embedded/bin/ruby
+#
+# Check Disk Plugin
+# ===
+#
+# Uses GNU's -T option for listing filesystem type; unfortunately, this
+# is not portable to BSD. Warning/critical levels are percentages only.
+#
+# Copyright 2011 Sonian, Inc <chefs@sonian.net>
```

Comment:
Where did we get this file from?

Please add info such as in https://github.com/blueboxgroup/ursula-monitoring/blob/master/sensu/plugins/metrics-process-usage.py#L21-L27 about where this came from, and use the proper license name.